### ISSUE TITLE
Pressing Ans in Sequences app should input uₙ

### ISSUE THREAD

**Issue by MDLC01** [numworks/epsilon#1757]
#### Problem you'd like to fix
When adding a recursive sequence, it can be annoying to open the toolbox and then select uₙ (or \<sequence name>ₙ).

#### Screenshots
![screenshot](https://user-images.githubusercontent.com/57839069/105982876-2fe89800-6098-11eb-934a-636eab131f2e.png)

#### Describe the solution you'd like
Currently, the Ans key is unused in the Sequence app. It would therefore be the best solution to this problem. Pressing Ans when defining a recursive sequence should input \<sequence name>ₙ.

#### Describe alternatives you've considered
I think Ans is the best solution, because it is currently unused, which means *it doesn't change any existing behavior*.

**Comment by Ecco** [comment]
That's an interesting suggestion, thanks @MDLC01 ! When considering that kind of modification, we usually consider two parts:
1. Could the new behavior be a regression (as in, a breaking change)
2. Can the new behavior be discovered / does it make sense

For 1, you're right, we're good! But I'm a bit concerned about 2: I'm not sure the average user could discover this feature on her own.

Also I'm not quite sure about the problem you're trying to solve: as you said, `Un` is right there in the toolbox!

If it's really that annoying to open the toolbox, an alternative could be to improve the behavior of the `x,n,t` key. The aim of that key, as the marking implies, is to offer values that makes sense from a contextual standpoint. And actually `Un` could be one of those.

**Comment by MDLC01** [comment]
I agree that the problem is not the most annoying one. But I just feel like it's a little weird to have to open a menu to input something as basic as `uₙ`.
As for using `x,n,t`, it could also work, but that key is already used to input `n` in this situation, and `Ans` felt like semantically more consistent in this case. However since you can input `n` by using `Alpha`, I think it's better to use `x,n,t` to input `uₙ` if you don't want to use `Ans` for that.
